### PR TITLE
Disable some package auto updates

### PR DESCRIPTION
**Pull request by huayang-codaio**
per https://github.com/coda/packs-sdk/pull/2644 and https://github.com/coda/packs-sdk/pull/2642

PTAL @dweitzman-codaio

### CODE REVIEW

File: renovate.json5 [hunk 1]
```diff
@@ -20,6 +20,18 @@
...
+    // pnpm 8.6.0 moves to a new lockfile version.
+    {
+      packageNames: ['pnpm'],
+      allowedVersions: '<8.6.0',
+    },
+
+    // see https://github.com/coda/packs-sdk/pull/2642
+    {
+      packageNames: ['open'],
+      allowedVersions: '8.4.2',
```
Comment (dweitzman-codaio) [inline renovate.json5:33]: Maybe make this <9.0.0 to allow for minor and patch releases?

**Review by dweitzman-codaio — APPROVED** [review]

File: renovate.json5 (hunk 1 above)
Comment (dweitzman-codaio) [reply renovate.json5:33]: Also, FYI I forced the off-schedule the creation of the PRs for these so I'd have a place to comment about them

I think closing the renovate PR for `open` will also discourage it from trying to mail PRs for the same major version in the future: https://docs.renovatebot.com/key-concepts/pull-requests/#normal-prs

Comment (huayang-codaio) [reply renovate.json5:33]: ah that's good to know. I was wondering how to control the renovate PRs.

Comment (dweitzman-codaio) [reply renovate.json5:33]: If you haven't seen it, there's also an issue at https://github.com/coda/packs-sdk/issues/2464 where you toggle a checkbox to force off-schedule PR creation or rebase a bunch of PRs at once

**Pull request merged**